crates/anvil/src/eth/api.rs

```diff
@@ -1917,6 +1926,120 @@ impl EthApi {
         Ok(())
     }
 
+    /// Reorg the chain by rewinding the state to a specific depth and mine new blocks ontop.
+    ///
+    /// Optionally supply a list of transactions that will populate the newly mined blocks
+    ///
+    /// Handler for RPC call: `anvil_reorg`
+    pub async fn anvil_reorg(
+        &self,
+        depth: u64,
+        new_len: u64,
```

Comment:
it's not obvious why we need both here

Reply:
Good callout. I was thinking that the reorged chain could be of any length. That is either less than or greater than the current canonical chain length. I'll fix this.